### DISCUSSION TITLE
Aligning prompt with survey-likert

### DISCUSSION THREAD

**Discussion by viviers** [jspsych/jsPsych#1036] [Q&A] [answered]
Right now the likert is shifted right slightly relative to the prompt text above it.

![image](https://user-images.githubusercontent.com/38060183/92589345-86725700-f2ee-11ea-85b4-746b4a595e95.png)

I have this css override in my html to ensure the text in all trials is in a relatively small column so that it's easier to read:
`    .jspsych-content {
      max-width: 45em;
    }`

Is this css causing the likert trial to display weirdly like this? If so - is there a way to center the prompt text relative to the likert without removing the css?

**Comment by viviers** [comment]
Update: I've figured out if I change ` .jspsych-content { max-width: 45em; }` to ` .jspsych-content { max-width: 95%; }` in devtools everything aligns perfectly. But I need it to be ` .jspsych-content { max-width: 45em; }`  in other trials... 

Is there any way to modify this basically global selector to have a `max-width` of 95% just for this one trial?

**Comment by becky-gilbert** [comment, marked as answer]
If you want to change the CSS for non-trial-specific elements on specific trials, one option might be to use the trial's [on_start](https://www.jspsych.org/overview/callbacks/#on_start-trial) or [on_load](https://www.jspsych.org/overview/callbacks/#on_load) function. In one of those functions you could access the element you want to change and then modify its style property, e.g.
```javascript
document.getElementsByClassName("jspsych-content")[0].style.maxWidth = "95%";
```
I think you'd also need to change it back at the end of the trial or start of the next trial.

**Reply by viviers** [reply]
Genius thank you so much!
